python/pdo/test/request.py

```diff
@@ -237,7 +255,7 @@ def CreateAndRegisterContract(config, enclave, contract_creator_keys) :
             logger.error('contract initialization failed: %s', initialize_response.result)
             ErrorShutdown()
 
-        contract.set_state(initialize_response.encrypted_state)
+        contract.set_state(initialize_response.raw_state)
```

Comment:
is this (encrypted_state -> raw_state) just a renaming ?

Reply:
yes.